developers/documentation/how_to_submit_pr_online.md

```diff
@@ -0,0 +1,111 @@
+---
+jupytext:
+  formats: ipynb,md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.10.3
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# How to open a pull request on GitHub for new documentation
+
+This how-to will guide you to open a pull request in GitHub to contribute your prepared document to napari.org. 
+
+This guide is strictly designed for adding new documents - if you want to contribute code to napari check out [this guide instead](../contributing.md).
+
+```{note}
+Don't yet a have a document ready? Check out our [main guide](./index.md) to find templates and learn how to prepare your document!
+```
+
+## Prerequisites
+
+- a new document for napari.org
+    - Your document should already be prepared for submission. Check out [our contributor's guide](./index.md) to learn how to prepare your document
+- a [GitHub account](https://github.com)
+
+## 1. Fork the napari repository
+
+Signed into your GitHub account, go to the [napari repository](https://github.com/napari/napari) and fork it to your own account by clicking the `Fork` button in the top right.
+
+![Screenshot of top right of napari's GitHub page with an arrow pointing to the Fork button](assets/fork_repo.png)
+
+## 2. Navigate to the right folder
+
+Click on folder names within the repository to navigate to the folder where your document needs to go.
+
+- **Explanations** go in `napari/docs/guides/`
+- **Tutorials** go in `napari/docs/tutorials/`
+- **How-tos** go in `napari/docs/howtos`
+
+Not sure where to place your document? Make a best guess and open the pull request - the napari team will
+help you edit your document and find the right spot!
+## 3. Upload your file
```

Comment:
This section needs to mention that we're asking people to upload a markdown file (so, not a `.ipynb` file). 

You may also need to link to the part of the docs that explains how to convert file formats using jupytext.